lib/modules/neverEndingReddit.js

```diff
@@ -102,7 +102,8 @@ module.go = function() {
 	siteTable.classList.add('res-ner-listing');
 
 	// modified from a contribution by Peter Siewert, thanks Peter!
-	const entries = document.body.querySelectorAll('a.comments');
+	// use div#siteTable in selector to avoid marking duplicates found in spotlight box
+	const entries = document.body.querySelectorAll('div#siteTable a.comments');
```

Comment:
Thanks!  Fixed.

On Fri, Jun 10, 2016 at 5:00 PM, Erik Desjardins <notifications@github.com>
wrote:

> In lib/modules/neverEndingReddit.js
> <https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3059#discussion_r66680343>
> :
>
> > @@ -102,7 +102,8 @@ module.go = function() {
> >  	siteTable.classList.add('res-ner-listing');
> >
> >  	// modified from a contribution by Peter Siewert, thanks Peter!
> > -	const entries = document.body.querySelectorAll('a.comments');
> > +	// use div#siteTable in selector to avoid marking duplicates found in spotlight box
> > +	const entries = document.body.querySelectorAll('div#siteTable a.comments');
>
> Nit: no div, just #siteTable a.comments
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3059/files/d233dc7210031acacb91004351bf579e8c89e17b#r66680343>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMQaKxv5CFbFN_J8WtgkiIQNCPwDIRT3ks5qKdCKgaJpZM4IycdW>
> .
>